lib/index.js

```diff
@@ -251,11 +252,12 @@ class Service extends AdapterService {
         [`${this.table}.${this.id}`]: { $in: idList }
       };
       const q = this.knexify(this.db(params), query);
+      const originalQuerySubset = params.query && params.query.$select ? { $select: params.query.$select } : {};
       const findParams = Object.assign({}, params, {
-        query: Object.assign({}, params.query, query)
+        query: Object.assign(originalQuerySubset, query)
```

Comment:
Without this change, the adapter test `.patch multi query changed` fails, because the original query is `{ age: 10 }`, but we patched age to 2 in that test, which results in not fetching any result.

At first, I wanted to introduce `returning` arg to `q.update`, like it's done in `create/insert` case, but that doesn't work for sqlite. So I've modified the code to only apply the $select from the original query for filtering the result set, but keep all other query bits out, so that we always reliably return patched objects (we don't want to do any filtering).

This does seem a bit of a backwards incompatible change, especially if someone was relying on this behaviour before (e.g. expecting nothing to be returned due to filtering, or using some other special query param that I haven't taken into account). But this is what was needed (or something like it) to pass the adapter tests.